flux-middle/src/rty/mod.rs

```diff
@@ -59,10 +59,17 @@ pub use crate::{
 #[derive(Debug, Clone)]
 pub struct Generics {
     pub params: List<GenericParamDef>,
+    pub refine_params: Vec<RefineParam>,
```

Comment:
We `Generics` to be cheaply clonable so we need to make this a `List` 

```suggestion
    pub refine_params: List<RefineParam>,
```